legal-test-fixture/legal_api/api/converter/JsonConverter.py

```diff
@@ -0,0 +1,97 @@
+from flask import json, jsonify
+from legal_api.models.business import Business, Director
+import logging
+from legal_api.api.converter.utils import format_date, format_non_date, format_boolean, format_json
+
+
+class JsonConverter():
+
+    def convert_to_json(self, business_list):
+        json_list = []
+
+        # convert each business to JSON
+        for business in business_list:
+            json_list.append(self.__json_business(business))
+
+        # return a single JSON object
+        return jsonify({'businesses': json_list})
+
+    def __json_business(self, business):
+        mailing_address = business.mailing_address.one_or_none()
```

Comment:
As per another comment, business address structure is changing.

Reply:
Resolving on new PR (rebasing is fubared)